platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -126,6 +126,33 @@ See the User Manual section on the “[Feature Lifecycle](userguide/feature_life
 
 The following are the features that have been promoted in this Gradle release.
 
+This Gradle release promotes the following features to stable:
```

Comment:
:x: this line says basically the same as the previous line

Also, let's put the promotion under a subtitle for the file permission api.